ports/geckolib/properties.mako.rs

```diff
@@ -209,7 +209,12 @@ impl ${style_struct.trait_name} for ${style_struct.gecko_struct_name} {
         // FIXME(bholley): Align binary representations and ditch |match| for cast + static_asserts
         self.gecko.${longhand.gecko_ffi_name} = match v {
             % for value in longhand.keyword.values_for('gecko'):
-                Keyword::${to_rust_ident(value)} => gss::${longhand.keyword.gecko_constant(value)} as u8,
+                % if value == 'full-width':
+            // FIXME: Deal with properties that deviate from naming conventions
```

Comment:
Rather than adding this here, I'd prefer to just blacklist the property by adding it to force_stub in the "Inconsistent constant naming in gecko" section above. Then I can fix both of them at the same time in gecko and remove that line from the blacklist.